### ISSUE TITLE
QCF font issue for 33:37 on firefox

### ISSUE THREAD

**Issue by naveed-ahmad** [quran/quran.com-frontend-v2#474]
و كان أمر الله
not
و كان أمر لله
https://quran.com/33/37

**Comment by Yuniac** [comment]
It is correct now? or I'm missing something?

![image](https://user-images.githubusercontent.com/76709894/129117971-b2c3fbfe-9ad3-4c4d-86ca-a8991b1b6ac1.png)

**Comment by naveed-ahmad** [comment]
No, still have this issue on Firefox. Working fine on Chrome and Safari.

**Comment by Yuniac** [comment]
> 
> 
> No, still have this issue on Firefox. Working fine on Chrome and Safari.

Interesting, I tried on Firefox (and the screenshot I posted was from Firefox) and it seemed fine

**Comment by naveed-ahmad** [comment]
Weird, can you share your OS version and Firefox version? 

Someone reported this and I can reproduce this in:
Mac 11.2.3
Firefox 91.0 (64-bit)

**Comment by Yuniac** [comment]
Sure;

Firefox: 91.0 (64-bit)
Windows 10: 10.0.19043 Build 19043

And I checked again right now and it is fine, the wording is correct.
